### PR TITLE
Yevgeny/mgmt 1643 new

### PR DESCRIPTION
**Pull request by yevgeny-shnaidman**

### CODE REVIEW

File: tools/utils.py [hunk 1]
```diff
@@ -17,23 +17,25 @@ def check_output(cmd):
 def get_service_host(service, target=None, domain="", namespace='assisted-installer'):
     if target is None or target == "minikube":
         reply = check_output("{} -n {} service --url {}".format(MINIKUBE_CMD, namespace, service))
-        return re.sub("http://(.*):.*", r'\1', reply)
+        host = re.sub("http://(.*):.*", r'\1', reply)
     elif target == "oc-ingress":
-        return "{}.{}".format(service, get_domain(domain, namespace))
+        host "{}.{}".format(service, get_domain(domain, namespace))
```
Comment (filanov) [inline tools/utils.py:22]: ?

File: tools/utils.py [hunk 2]
```diff
@@ -17,23 +17,25 @@ def check_output(cmd):
...
     if target is None or target == "minikube":
         reply = check_output("{} -n {} service --url {}".format(MINIKUBE_CMD, namespace, service))
-        return reply.split(":")[-1]
+        port = reply.split(":")[-1]
     else:
         cmd = '{kubecmd} -n {ns} get service {service} | grep {service}'.format(kubecmd=KUBECTL_CMD, ns=namespace, service=service)
         reply = check_output(cmd)[:-1].split()
-        return reply[4].split(":")[0]
+        port = reply[4].split(":")[0]
+    return port.strip()
```
Comment (filanov) [inline tools/utils.py:38]: split will return a list

File: tools/utils.py [hunk 3]
```diff
@@ -17,23 +17,25 @@ def check_output(cmd):
...
-        return re.sub("http://(.*):.*", r'\1', reply)
+        host = re.sub("http://(.*):.*", r'\1', reply)
     elif target == "oc-ingress":
-        return "{}.{}".format(service, get_domain(domain, namespace))
+        host "{}.{}".format(service, get_domain(domain, namespace))
     else:
         cmd = '{kubecmd} -n {ns} get service {service} | grep {service}'.format(kubecmd=KUBECTL_CMD, ns=namespace, service=service)
         reply = check_output(cmd)[:-1].split()
-        return reply[3]
+        host reply[3]
```
Comment (filanov) [inline tools/utils.py:26]: ?

File: internal/bminventory/inventory.go [hunk 4]
```diff
@@ -1597,7 +1597,7 @@ func (b *bareMetalInventory) createKubeconfigJob(cluster *common.Cluster, jobNam
 								},
 								{
 									Name:  "INVENTORY_ENDPOINT",
-									Value: "http://" + b.InventoryURL + ":" + b.InventoryPort + "/api/assisted-install/v1",
+									Value: "http://" + strings.TrimSpace(b.InventoryURL) + ":" + strings.TrimSpace(b.InventoryPort) + "/api/assisted-install/v1",
```
Comment (filanov) [inline internal/bminventory/inventory.go:1600]: `InventoryURL` and `InventoryPort` are used in other places as well

File: tools/utils.py (hunk 1 above)
Comment (eranco74) [inline tools/utils.py:22]:
```suggestion
        host = "{}.{}".format(service, get_domain(domain, namespace))
```

File: tools/utils.py (hunk 3 above)
Comment (eranco74) [inline tools/utils.py:26]:
```suggestion
        host = reply[3]
```

**Pull request merged**